examples/udpmulticat.c

```diff
@@ -75,34 +77,125 @@
 #include <upipe-modules/upipe_genaux.h>
 #include <upipe-modules/upipe_dup.h>
 #include <upipe-modules/upipe_udp_source.h>
+#include <upipe-modules/upipe_rtp_feedback.h>
+#include <upipe-modules/upipe_file_source.h>
 #include <upipe-modules/upipe_file_sink.h>
 #include <upipe-modules/upipe_multicat_sink.h>
 #include <upipe-modules/upipe_udp_sink.h>
+#include <upipe-modules/upipe_probe_uref.h>
 
 #include <stdbool.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <unistd.h>
 #include <inttypes.h>
 #include <signal.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+#include <ev.h>
 
 #define UDICT_POOL_DEPTH 10
 #define UREF_POOL_DEPTH 10
 #define UBUF_POOL_DEPTH 10
 #define UPUMP_POOL 10
 #define UPUMP_BLOCKER_POOL 10
 #define READ_SIZE 4096
-#define UPROBE_LOG_LEVEL UPROBE_LOG_WARNING
+
+static enum uprobe_log_level loglevel = UPROBE_LOG_DEBUG;
+
+static struct upipe *upipe_rtpfb;
+static struct upipe *upipe_rtpfb_sub;
+static struct upipe *upipe_udpsrc;
+static struct upipe *rtcp_sink;
+static const char *srcpath;
+
+static int udp_fd = -1;
+
+static struct sockaddr_storage addr;
+static socklen_t addr_len;
 
 static void usage(const char *argv0) {
-    fprintf(stdout, "Usage: %s [-d] [-r <rotate>] [-O <rotate offset>] <udp source> <dest dir/prefix> [<suffix>]\n", argv0);
+    fprintf(stdout, "Usage: %s [-d]  <udp source> <dest dir/prefix> <latency>", argv0);
     fprintf(stdout, "   -d: force debug log level\n");
-    fprintf(stdout, "   -r: rotate interval in 27MHz unit\n");
```

Comment:
Why do you remove features from the examples? If it's too hard to have both use cases in the same example, create a new one.

Reply:
IMO it's always interesting to have an example

Reply:
@kierank Which license for the examples?